Library/Homebrew/manpages/brew.1.md

```diff
@@ -126,6 +126,25 @@ Note that these flags should only appear after a command.
     type dependencies, pass `--skip-build`. Similarly, pass `--skip-optional`
     to skip `:optional` dependencies.
 
+  * `desc` <formula>:
+    Display <formula>'s name and one-line description.
+
+  * `desc [-s|-n|-d] <pattern>`:
+    Search both name and description (`-s`), just the names (`-n`), or just  the
+    descriptions (`-d`) for `<pattern>`. `<pattern>` is by default interpreted
+    as a literal string; if flanked by slashes, it is instead interpreted as a
+    regular expression. Formula descriptions are cached, and the cache is
+    created on the first search, making that search slower than subsequent
+    ones.
+
+  * `desc --create-cache`:
+
+    Force the [re]construction of the name/description cache.
+
+  * `desc --delete-cache`:
+
+    Delete the name/description cache, if it exists.
```

Comment:
I don't think we need to document both these options (or possibly have them at all, certainly `delete-cache`